zcash_client_backend/src/scan.rs

```diff
@@ -1,200 +1,19 @@
-use core::fmt;
+use crossbeam_channel as channel;
 use std::collections::HashMap;
-use std::io;
+use std::fmt;
 use std::mem;
 
-use crossbeam_channel as channel;
```

Comment:
To @daira's point, this change in the second commit would not be present if the split between `core/std` imports and external crate imports had been preserved.